agenthub/dummy_agent/agent.py

```diff
@@ -44,9 +45,15 @@ class DummyAgent(Agent):
     without making any LLM calls.
     """
 
+    runtime_tools: list[RuntimeTool] = [RuntimeTool.BROWSER]
```

Comment:
Is this in use?

Reply:
Yes. BrowseURLAction and BrowseInteractiveAction are tested here.